cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/ssa0/SsaInternals.qll

```diff
@@ -302,9 +302,9 @@ class Def extends DefOrUse {
 
   Instruction getAddress() { result = this.getAddressOperand().getDef() }
 
-  Instruction getDefiningInstruction() { result = defOrUse.getDefiningInstruction() }
+  Node0Impl getValue() { result = defOrUse.getValue() }
 
-  override string toString() { result = this.asDefOrUse().toString() + " (def)" }
```

Comment:
Is there a particular reason why ` (def)` is dropped?

Reply:
Oh, well spotted. I realized that the `toString` in the `ssa0/SsaInternals.qll` file was different from the `toString` `SsaInternals.qll` file. So I just ninja-fixed this. This doesn't really matter as these `toString`s are only used during debugging.

Eventually, these should be ported to a parameterized module. This should be relatively straightforward to do. But I'm patiently waiting for the ability to quick-eval inside parameterized modules before I do that rewrite 😄.